packages/fx-core/src/plugins/solution/fx-solution/debug/util/tasksTransparency.ts

```diff
@@ -378,13 +378,13 @@ function installAzureFunctionsBindingExtensions(): CommentJSONValue {
 }
 
 function startLocalTunnel(): Record<string, unknown> {
+  // TODO: add comment
   return {
     label: "Start local tunnel",
     type: "teamsfx",
     command: "debug-start-local-tunnel",
     args: {
-      configFile: ".fx/configs/ngrok.yml",
-      keepAlive: false,
+      ngrokArgs: "http 3978",
```

Comment:
By default add `log` and `log-format`?

Reply:
> By default add `log` and `log-format`?

If args do not contains `--log` or `--log-format`, by default added them.

What about hidden the log args to make the `ngrokArgs` simple?

Reply:
For most users, keep the args consistent between tasks.json and terminal.
It may confuse user that he/she sees `http 3978` in tasks.json, but `http 3978 --log...` in terminal.